src/views/Workflow.vue

```diff
@@ -93,15 +105,30 @@ export default {
   },
   beforeRouteEnter (to, from, next) {
     next(vm => {
-      // Create a Tree View for the current workflow by default
-      const subscriptionId = vm.subscribe('tree')
       vm.$nextTick(() => {
+        vm.subscribeToDeltas()
+        // Create a Tree View for the current workflow by default
+        const subscriptionId = new Date().getTime()
```

Comment:
Is this necessary if the Tree Component initiates its own subscriptions?

Reply:
That's because the Lumino widgets require an ID as argument when created.

Reply:
Ah, sorry I was commenting more on the `vm.subscribeToDeltas()` bit.

Reply:
Ah, that one. That's still necessary to start the GraphQL deltas subscription. Before, we were also calling `register`/`susbcribe`. Now we just `subscribeToDeltas`, which will tell `WorkflowService` to start the query and keep track of the subscription.

As there are no merging here, we have a single deltas subscription, so we don't need the ID as before.

Reply:
Re-reading the code @oliver-sanders , I realized now that the deltas subscription is being created when the first widget is added, and then never removing it. Let me see if I can find an alternative for this.

Reply:
Any luck?

Reply:
Ah that should be fixed (checking again now). I got distracted after I wrote this comment with the issue in the order of deltas messages and forgot to update this comment. Sorry.

Reply:
It was closing the deltas subscription once you closed any tree widget. Added a check to confirm there are no more tree widgets before closing it. Should be good to go. Tested with a workflow running, and looking at the network tab in the browser. Confirmed there were only 2 subscription (gscan, deltas) and once all tree widgets were closed, the deltas is stopped. Then adding a tree widget, a new deltas subscription is created.